cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsHandlerTestRun.java

```diff
@@ -168,11 +174,15 @@ public void testSearchWithTags() throws Exception {
     // WCount should be found in yourspace, not in myspace
     verifySearchResultWithTags("/v3/metrics/search?target=tag&tag=namespace:yourspace&tag=app:WCount1",
                        getSearchResultExpected("flow", "WCounter",
-                                                "flow", "WordCounter",
-                                                "mapreduce", "ClassicWordCount",
-                                                "procedure", "RCounts"
+                                               "flow", "WordCounter",
+                                               "mapreduce", "ClassicWordCount",
+                                               "procedure", "RCounts"
                        ));
 
+    // Child context for adapters should be empty
```

Comment:
we don't use child context terminology anymore, so just say, no tags under adapter